src/components/hand-poses.js

```diff
@@ -28,9 +46,11 @@ AFRAME.registerComponent("hand-pose", {
     this.from.play();
   },
 
-  update(oldData) {
-    if (oldData.pose != this.data.pose) {
-      this.animatePose(NETWORK_POSES[oldData.pose || 0], NETWORK_POSES[this.data.pose]);
+  tick() {
+    if (!this.store) return;
+    if (this.store.data.pose != this.pose) {
```

Comment:
`!==` ?

Reply:
Also I'd probably just do:
`if (!this.store || !this.store.data || this.store.data.pose !== this.pose) return;`

Reply:
👍